### ISSUE TITLE
Disable indexes not synced error

### ISSUE THREAD

**Issue by fishcharlie** [dynamoosejs/dynamoose#294]
I have the following schema/models in Dynamoose in my cron job code.

```
var Schema = new dynamoose.Schema({_id: String}, {
	saveUnknown: true
});
var UserModel = dynamoose.model('User', Schema);
var ItemModel = dynamoose.model('Item', Schema);
```

Problem is, in my main application the schema I use for the `User` model has an index on one of the properties. But not the item.

In my cron job I would like to keep my schema's and models as simple as possible as I might forget to change it in the cron when I change it in my main application.

Therefor I'm getting the following error.

```
Error: indexes are not synchronized and update flag is set to false
    at /home/ubuntu/node/node_modules/dynamoose/lib/Table.js:166:29
    at _fulfilled (/home/ubuntu/node/node_modules/q/q.js:854:54)
    at self.promiseDispatch.done (/home/ubuntu/node/node_modules/q/q.js:883:30)
    at Promise.promise.promiseDispatch (/home/ubuntu/node/node_modules/q/q.js:8$
    at /home/ubuntu/node/node_modules/q/q.js:624:44
    at runSingle (/home/ubuntu/node/node_modules/q/q.js:137:13)
    at flush (/home/ubuntu/node/node_modules/q/q.js:125:13)
    at _combinedTickCallback (internal/process/next_tick.js:131:7)
    at process._tickDomainCallback (internal/process/next_tick.js:218:9)

```

Is there anyway to disable this error? Is it something that can even be disabled or is it integrated tightly into the AWS SDK?

**Comment by tsdorsey** [comment]
tl;dr. `var UserModel = dynamoose.model('User', Schema, {create: false});`

@fishcharlie I was running into this as well. However the "check indexes" code is inside an `if(model.options.create)` check so if you disable table creation (which is on by default) the errors about indexes will stop.

I only use dynamoose to access already existing tables so I created a helper function to set options on the schema and model the same way every time. Here it is if you want it.
```
export function model(tableName, fields, statics, methods) {
  let schema = new dynamoose.Schema(fields, { timestamps: true });
  schema.statics = statics;
  schema.methods = methods;

  return dynamoose.model(tableName, schema, {
    create: false,
    update: false,
  });
}```
